src/Sylius/Component/Core/Model/ChannelPricing.php

```diff
@@ -24,7 +22,7 @@ class ChannelPricing implements ChannelPricingInterface
     protected $id;
 
     /**
-     * @var ChannelInterface
+     * @var string
      */
     protected $channel;
```

Comment:
fair enough for me, we can stay with the channel. But on the other hand on an order entity we have currency code and locale code. Just sayin' ;)